### PR TITLE
[Enhancement] Add some logs for MetastoreEvenetProcessor

### PR DESCRIPTION
**Pull request by stephen-shelby**
## What type of PR is this：
- [ ] bugfix
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 0 / 20 (00.00%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/external/hive/events/MetastoreEventFactory.java|0|5|00.00%|[66, 69, 87, 88, 99]|
|:large_blue_circle:|com/starrocks/external/hive/events/MetastoreEventsProcessor.java|0|8|00.00%|[212, 215, 223, 230, 237, 350, 366, 373]|
|:large_blue_circle:|com/starrocks/external/hive/events/AddPartitionEvent.java|0|2|00.00%|[108, 109]|
|:large_blue_circle:|com/starrocks/external/hive/events/AlterPartitionEvent.java|0|3|00.00%|[85, 86, 87]|
|:large_blue_circle:|com/starrocks/external/hive/events/AlterTableEvent.java|0|2|00.00%|[114, 115]|

**Review by caneGuy — APPROVED** [review]

File: fe/fe-core/src/main/java/com/starrocks/external/hive/events/AddPartitionEvent.java [hunk 1]
```diff
@@ -105,6 +105,8 @@ protected boolean isSupported() {
     @Override
     protected void process() throws MetastoreNotificationException {
         if (!existInCache()) {
+            LOG.warn("Table [{}.{}.{}] doesn't exist in cache on event id: [{}]",
```
Comment (DorianZheng) [inline fe/fe-core/src/main/java/com/starrocks/external/hive/events/AddPartitionEvent.java:108]: Will this print too much log? And `info` level might be suitable.

**Review by dirtysalt — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: fe/fe-core/src/main/java/com/starrocks/external/hive/events/AddPartitionEvent.java (hunk 1 above)
Comment (dirtysalt) [reply fe/fe-core/src/main/java/com/starrocks/external/hive/events/AddPartitionEvent.java:108]: `warn` produces less logs.

**Comment by stephen-shelby** [comment]
run starrocks_admit_test

**Comment by stephen-shelby** [comment]
run starrocks_admit_test

**Comment by stephen-shelby** [comment]
run starrocks_admit_test

**Comment by stephen-shelby** [comment]
run starrocks_admit_test

**Comment by stephen-shelby** [comment]
https://github.com/Mergifyio backport branch-2.4

**Pull request merged**